### DISCUSSION TITLE
Can't approve my student document

### DISCUSSION THREAD

**Discussion by sahnomer** [community/community#129847] [GitHub Education]
### Hi everyone,

I had another account which I have approved my student discount on and for some reasons I had to delete that account and create a new one which now I am posting this from. I don't know why but now I can not upload my proof on the github education application. I am trying to upload my document and click on the "process" button but it does nothing but just refreshing the website. Why I am not able to benefit from my studentship :( please someone help me prove it

**Comment by davevad93** [comment]
Hi @sahnomer ,

You can try the following:

1. This user was able to submit the application by taking a photo from the webcam: #126276 
2. It could be a cache related issue, this user solved it by switching to another browser: [discussioncomment-9853042](https://github.com/orgs/community/discussions/129399#discussioncomment-9853042)
